### PR TITLE
feat: COR-1486-toggle-vr-references-removal

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard27**
## Summary

- Removed toggle button, onChartRegionChange, vr DynamicChoropleth from [rioolwater.tsx](https://github.com/minvws/nl-covid19-data-dashboard/compare/epic/COR-1475-safety-region-removal...feature/COR-1486-rioolwater-toggle-button-removal?expand=1#diff-b322aae36ff7b99d9d568a24d5d30ab3fe789031645c83472a2b5e4d61bf8429)

## Screenshots
**_Before_**
<details>

  <summary>Toggle before screenshot</summary> 
Screenshot 2023-03-27 at 17 14 37

</details>

**_After_**
<details>
  <summary>Toggle after screenshot</summary> 

Screenshot 2023-03-27 at 17 14 00

</details>

### CODE REVIEW

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Great job!

File: packages/app/src/pages/landelijk/rioolwater.tsx [hunk 1]
```diff
@@ -146,12 +142,6 @@ const SewerWater = (props: StaticProps<typeof getStaticProps>) => {
           <ChoroplethTile
             title={textNl.map_titel}
             description={textNl.map_toelichting}
-            metadata={{
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/pages/landelijk/rioolwater.tsx:149]: I expect that the metadata is still needed here.

**Review by VWSCoronaDashboard18 — COMMENTED** [review]
Maybe you can also adjust the VR threshold on line 152.

*edit: it's been updated!

File: packages/app/src/pages/landelijk/rioolwater.tsx (hunk 1 above)
Comment (VWSCoronaDashboard27) [reply packages/app/src/pages/landelijk/rioolwater.tsx:149]: @VWSCoronaDashboard21  I have applied the changes, please fill free to resolve.

File: packages/app/src/pages/landelijk/rioolwater.tsx [hunk 2]
```diff
@@ -147,58 +143,34 @@ const SewerWater = (props: StaticProps<typeof getStaticProps>) => {
...
-                  getLink: reverseRouter.gm.rioolwater,
-                }}
-              />
-            )}
-
-            {selectedMap === 'vr' && (
-              <DynamicChoropleth
-                map="vr"
-                accessibility={{
-                  key: 'sewer_region_choropleth',
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard27) [reply packages/app/src/pages/landelijk/rioolwater.tsx:187]: @VWSCoronaDashboard18  Done

**Review by VWSCoronaDashboard26 — APPROVED** [review]

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Looks great! Good job.

File: packages/app/src/pages/landelijk/rioolwater.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard18) [reply packages/app/src/pages/landelijk/rioolwater.tsx:187]: Nice, I'll resolve this!

**Pull request merged**